minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/RunMiNiFi.java

```diff
@@ -357,8 +387,10 @@ private synchronized void saveProperties(final Properties nifiProps, final Logge
 
         try {
             final Set<PosixFilePermission> perms = new HashSet<>();
-            perms.add(PosixFilePermission.OWNER_READ);
             perms.add(PosixFilePermission.OWNER_WRITE);
+            perms.add(PosixFilePermission.OWNER_READ);
+            perms.add(PosixFilePermission.GROUP_READ);
+            perms.add(PosixFilePermission.OTHERS_READ);
```

Comment:
@JPercivall maybe not relevant in MiNiFi context but when I see "others" getting permissions I always wonder why it's necessary and if any sensitive data can leak.